src/utils.rs

```diff
@@ -118,53 +135,69 @@ pub fn check_for_config() -> Result<bool, anyhow::Error> {
 
     if !bedrust_config_file_path.exists() || !model_config_file_path.exists() {
         Ok(false)
-    } else { 
+    } else {
         Ok(true)
     }
+}
+
+pub fn prompt_for_model_selection() -> Result<ArgModels, anyhow::Error> {
+    let model_list = ArgModels::value_variants();
+    let idx = FuzzySelect::with_theme(&ColorfulTheme::default())
+        .with_prompt("Select a default model to use or <enter> to skip")
+        .items(model_list)
+        .interact()?;
+    Ok(model_list[idx])
+}
 
+pub fn prompt_for_model_selection_opt() -> Result<Option<ArgModels>, anyhow::Error> {
+    let model_list = ArgModels::value_variants();
+    let idx = FuzzySelect::with_theme(&ColorfulTheme::default())
+        .with_prompt("Select a default model to use or <enter> to skip")
+        .items(model_list)
+        .interact_opt()?;
+    Ok(idx.map(|idx|model_list[idx]))
 }
 
 // function that creates the configuration files during the `init` command
-pub fn initialize_config() -> Result<(), anyhow::Error>{
+pub fn initialize_config() -> Result<(), anyhow::Error> {
     let home_dir = home_dir().expect("Failed to get HOME directory");
     let config_dir = home_dir.join(format!(".config/{}", constants::CONFIG_DIR_NAME));
     fs::create_dir_all(&config_dir)?;
 
     let bedrust_config_file_path = config_dir.join(constants::BEDRUST_CONFIG_FILE_NAME);
     let bedrust_config_content = constants::BEDRUST_CONFIG_FILE.to_string();
-    fs::write(&bedrust_config_file_path, bedrust_config_content)?;
-    println!("⏳| Bedrust configuration file created at: {:?}", bedrust_config_file_path);
+
+    let mut default_config: BedrustConfig =
+        ron::de::from_str(&bedrust_config_content).expect("default config must be valid");
+    default_config.default_model = prompt_for_model_selection_opt()?;
+
+    fs::write(
+        &bedrust_config_file_path,
+        ron::ser::to_string_pretty(&default_config, PrettyConfig::new())?,
+    )?;
+    println!(
+        "⏳| Bedrust configuration file created at: {:?}",
+        bedrust_config_file_path
```

Comment:
for the moment, I reparsed the config. Probably would be nice to store the config in struct form instead eventually.